### DISCUSSION TITLE
Custom backup inclusions for dediconnect.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#450] [Suggestions]
There are important files not backed up by the default system. It would be great to have some ability to include these in backups on Dediconnect machines.

Among these are mods like TC Auto Rewards database. Dino Storage's recovery file. (Stored under SaveGames )

This also includes sqlite databases for plugins. (Ark Homes for one)

**Comment by mvaessen** [comment]
Hi,

The SaveGames for ARK should currently be included in the backups.

Would you mind posting some screenshots about the SQLite db's? That should help better understand what kind of files need to be included.

I've removed the part of your suggestions about backing up cluster data/uploads. The cluster files should not be included in backups for individual game servers. Doing so can cause a lot of issues when rolling back. Feel free to post a separate suggestion for cluster files, if you like to see that added.
